syntax = "proto3";

package gis;

service GIS {
    rpc GetParcelInfo(ParcelRequest) returns (ParcelData) {}
}

message ParcelRequest {
    string pin_number = 1;
    string county = 2;
    // Two letter abbreviation of the state
    string state = 3;
}

message ParcelData {
    string pin_number = 1;
    string owner_name = 2;
    double acreage = 3;
    string street_address = 4;
    // This should be a WKT string in WGS84 (lat/long) projection
    string boundary_wkt = 5;
}
